### ISSUE TITLE
Feature: Classify with minimum threshold 

### ISSUE THREAD

**Issue by Ch4s3** [jekyll/classifier-reborn#22]
We monkey patched ClassifierReborn::Bayes with a method that allows you to classify text but return "Not Found" if the classifications method returns a score below a given threshold. The idea is to cut down on false positives.

Would this be an interesting feature to include in the gem proper? If so I'd be happy to tackle that.

**Comment by inspire22** [comment]
Checking up, I would love to see customizable thresholding...

**Comment by Ch4s3** [comment]
Since you asked, I'll try to get it done this weekend. @inspire22, I'll base it off of the code my company uses to set the threshold, but do you have any requests or special use case in mind? I'd be happy to incorporate feedback.

**Comment by inspire22** [comment]
Awesome, thanks :)

I'm using to automatically categories stuff into "featured" and "not featured".  I figured there'd be some kind of internal threshold such that if I have 100,000 items that I'm sorting through, I could get varying numbers of documents returning "featured".  I figured it'd be a 0-1 number that would affect it increasingly.  I haven't looked at the bayes algorithm in a long time though so it's possible this doesn't make any sense at all..
